fv3core/stencils/fxadv.py

```diff
@@ -368,117 +338,155 @@ def fxadv_fluxes_stencil(
     with computation(PARALLEL), interval(...):
         prod = dt * ut
         with horizontal(region[local_is : local_ie + 2, :]):
-            crx_adv = prod * rdxa[-1, 0] if prod > 0 else prod * rdxa
-            xfx_adv = dy * prod * sin_sg3[-1, 0] if prod > 0 else dy * prod * sin_sg1
+            crx = prod * rdxa[-1, 0] if prod > 0 else prod * rdxa
+            x_area_flux = (
+                dy * prod * sin_sg3[-1, 0] if prod > 0 else dy * prod * sin_sg1
+            )
```

Comment:
Minor:
``` if prod>0:
        crx = prod * rdxa[-1, 0]
        x_area_flux = dy * prod * sin_sg3[-1, 0]
     else:
        crx = prod * rdxa
        x_area_flux = dy * prod * sin_sg1
```
Depending on the codegen that will result in (slightly) less thread-split. Likewise below.

Reply:
thanks! yeah this was written back in the days when we *only* could do terniary if statements :)